### PR TITLE
remove the incorrect cluster_id from status.managed_cluster

### PR DESCRIPTION
**Pull request by yanmxa**
1. remove the default `uid` when inserting the record to `status.managed_cluster`, using the `cluster_id` from `clusterclaim` as the source of truth.
2. add triggers on `status.managed_cluster` when inserting to set the `cluster_id` in the `status.compliance` and `status.local_compliance `

### CODE REVIEW

File: operator/pkg/controllers/hubofhubs/database/4.trigger.sql [hunk 1]
```diff
@@ -66,7 +66,20 @@ CREATE TRIGGER set_timestamp BEFORE UPDATE ON spec.policies FOR EACH ROW EXECUTE
 DROP TRIGGER IF EXISTS set_timestamp ON spec.subscriptions;
 CREATE TRIGGER set_timestamp BEFORE UPDATE ON spec.subscriptions FOR EACH ROW EXECUTE FUNCTION public.trigger_set_timestamp();
 
+-- set the cluster_id to compliance table
 DROP TRIGGER IF EXISTS update_compliance_table ON local_status.compliance;
 CREATE TRIGGER update_compliance_table AFTER INSERT OR UPDATE ON local_status.compliance FOR EACH ROW WHEN (pg_trigger_depth() < 1) EXECUTE FUNCTION public.set_cluster_id_to_local_compliance();
+
 DROP TRIGGER IF EXISTS update_compliance_table ON status.compliance;
 CREATE TRIGGER update_compliance_table AFTER INSERT OR UPDATE ON status.compliance FOR EACH ROW WHEN (pg_trigger_depth() < 1) EXECUTE FUNCTION public.set_cluster_id_to_compliance();
```
[the remark this answers is not in the document]
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/database/4.trigger.sql:74]: @clyang82 Typo

[the remark this answers is not in the document]
Comment (yanmxa) [reply operator/pkg/controllers/hubofhubs/database/4.trigger.sql:74]: @clyang82 This is the trigger on the compliance table.
I think we discussed adding the insert trigger to the `status.managed_clusters` table~

**Review by clyang82 — APPROVED** [review]
LGTM

**Pull request merged**